test/dates/types.jl

```diff
@@ -168,6 +192,12 @@ ms = Dates.Millisecond(1)
 @test Dates.Date(d,y) == Dates.Date(1,1,1)
 @test Dates.Date(d,m) == Dates.Date(1,1,1)
 @test Dates.Date(m,y) == Dates.Date(1,1,1)
-
-@test isfinite(Dates.Date)
-@test isfinite(Dates.DateTime)
+us = Dates.Microsecond(1)
+ns = Dates.Nanosecond(1)
+@test Dates.Time(h) == Dates.Time(1)
+@test Dates.Time(h,mi) == Dates.Time(1,1)
+@test Dates.Time(h,mi,s) == Dates.Time(1,1,1)
+@test Dates.Time(h,mi,s,ms) == Dates.Time(1,1,1,1)
+@test Dates.Time(h,mi,s,ms,us) == Dates.Time(1,1,1,1,1)
+@test Dates.Time(h,mi,s,ms,us,ns) == Dates.Time(1,1,1,1,1,1)
+@test Dates.Time(us,h,s,ns,mi,ms) == Dates.Time(1,1,1,1,1,1)
```

Comment:
As I mentioned, there are actually plenty of other tests that cover different inputs, such [here](https://github.com/JuliaLang/julia/blob/f263b4ae79aa2424df6aa7f50f6af058484c37e5/test/dates/types.jl#L54), and [here](https://github.com/JuliaLang/julia/blob/f263b4ae79aa2424df6aa7f50f6af058484c37e5/test/dates/types.jl#L112), as well as [here](https://github.com/JuliaLang/julia/blob/f263b4ae79aa2424df6aa7f50f6af058484c37e5/test/dates/accessors.jl#L113).